code/modules/mob/living/carbon/human/life.dm

```diff
@@ -741,7 +741,7 @@
 		return 0
 
 	//SSD check, if a logged player is awake put them back to sleep!
-	if(species.show_ssd && (!client && !vr_mob) && !teleop)
+	if(species.show_ssd && (!client && !vr_mob) && !teleop && ((world.realtime - disconnect_time) >= 5 MINUTES)) //only sleep after 5 minutes, should help those with intermittent internet connections
```

Comment:
30 seconds or 1 minute should be enough.

Reply:
5 minutes is to play it safe

especially with BYOND ads and whatnot